### ISSUE TITLE
[Accessibility] Insufficient Color Contrast - UI component

### ISSUE THREAD

**Issue by amberhinds** [woocommerce/woocommerce-gateway-amazon-pay#325]
## Audit Information
This issue came from an audit WooCommerce had Equalize Digital complete in December 2025. I'm moving the issue out of Linear (WooA11Y-417) to this repo.

- Audit ID: WooExt-512
- Component/Page: Cart
- Affected Element: Amazon Pay
- URL where issue was discovered: https://apay-woo.mystagingwebsite.com/cart/

## WCAG Compliance

- Success Criterion: 1.4.11 Non-text Contrast
- WCAG Version: 2.1
- Level: AA
- Severity: Medium

## Issue Description

The focus indicator modifies the Amazon Pay component’s border within its visible boundary, and it fails color contrast against the background color. See color contrast analysis: https://webaim.org/resources/contrastchecker/?fcolor=E77600&bcolor=F0C14D

### Impacted Populations

People with low vision

## Recommended Fix

Change the colors to have a minimum color contrast of at least 3:1 or remove the defined color on the focus indicator so it can be controlled by the browser.

## Code Example
```
.amazonpay-button-view1-gold:focus {
    outline: none;
    border-color: #e77600;
    box-shadow: 0 0 3px 2px rgba(228, 121, 17, 0.5);
}
```
## Screenshots

https://drive.google.com/file/d/1VpKBLW_J1Ry21A4pDIb3wWAaz9EGMMFp/view?usp=drivesdk

**Comment by ChristianAF** [comment]
SDK issue.
